invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -283,10 +283,11 @@ def new_record():
 # Views
 #
 @login_required
-def deposit_search():
-    """List of user deposits page."""
+def dashboard(dashboard_name=None):
+    """Display user dashboard page."""
     return render_template(
-        "invenio_app_rdm/records/search_deposit.html",
+        "invenio_app_rdm/records/dashboard.html",
+        dashboard_name=dashboard_name or "uploads",
```

Comment:
This `or "uploads"` was a temporary hack... I think that we could have a URL converter for the route `dashboard_specific` to accept only `[uploads|communities|requests]`.

Reply:
Ok, I haven't worked on it yet. I created https://github.com/inveniosoftware/invenio-app-rdm/issues/1184 so I or someone else can work on it independently!